Sources/RemoteNotification.swift

```diff
@@ -108,119 +136,26 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
-public func ==(lhs: RemoteNotification.Alert, rhs: RemoteNotification.Alert) -> Bool {
-    if lhs.body != nil || rhs.body != nil {
-        guard let lhsBody = lhs.body, let rhsBody = rhs.body where lhsBody == rhsBody else {
-            return false
-        }
-    }
-    
-    if lhs.bodyLocalizationKey != nil || rhs.bodyLocalizationKey != nil {
-        guard let lhsBodyLocalizationKey = lhs.bodyLocalizationKey, let rhsBodyLocalizationKey = rhs.bodyLocalizationKey where lhsBodyLocalizationKey == rhsBodyLocalizationKey else {
-            return false
-        }
-    }
-    
-    if lhs.bodyLocalizationArguments != nil || rhs.bodyLocalizationArguments != nil {
-        guard let lhsBodyLocalizationArguments = lhs.bodyLocalizationArguments, let rhsBodyLocalizationArguments = rhs.bodyLocalizationArguments where lhsBodyLocalizationArguments == rhsBodyLocalizationArguments else {
-            return false
-        }
-    }
-
-    if lhs.actionLocalizationKey != nil || rhs.actionLocalizationKey != nil {
-        guard let lhsActionLocalizationKey = lhs.actionLocalizationKey, let rhsActionLocalizationKey = rhs.actionLocalizationKey where lhsActionLocalizationKey == rhsActionLocalizationKey else {
-            return false
+            wearableTitle = alert[alertWearableTitleKey] as! String?
+            wearableTitleLocalizationKey = alert[alertWearableTitleLocKey] as! String?
+            wearableTitleLocalizationArguments = alert[alertWearableTitleLocArgsKey] as! [String]?
         }
     }
-    
-    if lhs.launchImageName != nil || rhs.launchImageName != nil {
-        guard let lhsLaunchImageName = lhs.launchImageName, let rhsLaunchImageName = rhs.launchImageName where lhsLaunchImageName == rhsLaunchImageName else {
-            return false
-        }
-    }
-    
-    if lhs.wearableTitle != nil || rhs.wearableTitle != nil {
-        guard let lhsWearableTitle = lhs.wearableTitle, let rhsWearableTitle = rhs.wearableTitle where lhsWearableTitle == rhsWearableTitle else {
-            return false
-        }
-    }
-    
-    if lhs.wearableTitleLocalizationKey != nil || rhs.wearableTitleLocalizationKey != nil {
-        guard let lhsWearableTitleLocalizationKey = lhs.wearableTitleLocalizationKey, let rhsWearableTitleLocalizationKey = rhs.wearableTitleLocalizationKey where lhsWearableTitleLocalizationKey == rhsWearableTitleLocalizationKey else {
-            return false
-        }
-    }
-    
-    if lhs.wearableTitleLocalizationArguments != nil || rhs.wearableTitleLocalizationArguments != nil {
-        guard let lhsWearableTitleLocalizationArguments = lhs.wearableTitleLocalizationArguments, let rhsWearableTitleLocalizationArguments = rhs.wearableTitleLocalizationArguments where lhsWearableTitleLocalizationArguments == rhsWearableTitleLocalizationArguments else {
-            return false
-        }
-    }
-    
-    return true
 }
 
 
 // MARK: Top Level APS Keys
 
 
-let APSServiceKey = "aps"
+let apsServiceKey = "aps"
```

Comment:
happy to go either way on this. I just changed it for @mikelikespie. I originally had kept it as capitalized because acronyms but don't have strong opinions.

@efirestone for a tie-breaking vote?